public/app/app.ts

```diff
@@ -90,6 +90,15 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 export class GrafanaApp {
+  storedLanguage: string;
+
+  constructor() {
+    // Check for Iiris language
+    this.storedLanguage = localStorage.getItem('iiris_language') || 'fi';
+    console.log('this.storedLanguage: ' + this.storedLanguage);
```

Comment:
Maybe omit console.log in production? :)

Reply:
Good point! In commit 6db155846910b1ad703c3b08d014957ced91d959.